### PR TITLE
Add tutorial how to install Shopware 6

### PR DESCRIPTION
**Pull request by shyim**
It's my first tutorial. I hope it has not soo many issues :)

### CODE REVIEW

File: tutorials/install-and-configure-shopware-6/01.en.md [hunk 1]
```diff
@@ -0,0 +1,277 @@
...
+date: "2020-05-19"
+title: "Installation and configuration of Shopware 6"
+short_description: "Installation and configuration of Shopware 6."
+tags: ["Shopware", "eCommerce", "PHP"]
+author: "shyim"
+author_link: "https://github.com/shyim"
+author_img: "https://avatars3.githubusercontent.com/u/6224096"
+author_description: ""
+language: "en"
+available_languages: ["en"]
```
Comment (n-se) [inline tutorials/install-and-configure-shopware-6/01.en.md:14]: Please do not delete tags ;) Our community manager will fill them for you if you dont know what to do with them
`header_img` is missing here

Comment (shyim) [reply tutorials/install-and-configure-shopware-6/01.en.md:14]: Can i cross-reference to https://community.hetzner.com/tutorials/install-and-secure-nginx-lets-encrypt-debian-10 ?

**Comment by hhoechtl** [comment]
@shyim aren't you missing the `scheduler:run` command?

File: tutorials/install-and-configure-shopware-6/01.en.md [hunk 2]
```diff
@@ -0,0 +1,307 @@
...
+
+After configuring MariaDB, PHP and Nginx we start the installation of Shopware. We will install it to `/var/www/html`.
+
+```
+cd /var/www/html
+
+# Remove default file in this folder
+sudo rm index.nginx-debian.html
+
+# Download Shopware 6
```
Comment (n-se) [inline tutorials/install-and-configure-shopware-6/01.en.md:170]: Can you please add a note that the reader of this tutorial should check for new shopware 6 versions? As long as there are no major releases (7) I think everything should be fine

Comment (shyim) [reply tutorials/install-and-configure-shopware-6/01.en.md]: I did added :)

File: tutorials/install-and-configure-shopware-6/01.en.md [hunk 3]
```diff
@@ -167,7 +167,7 @@ cd /var/www/html
 # Remove default file in this folder
 sudo rm index.nginx-debian.html
 
-# Download Shopware 6
+# Download Shopware 6 (see https://www.shopware.com/de/changelog/ for latest versions)
```
Comment (n-se) [inline tutorials/install-and-configure-shopware-6/01.en.md:170]:
```suggestion
# Download Shopware 6 (see https://www.shopware.com/en/changelog/ for latest versions)
```

Comment (shyim) [reply tutorials/install-and-configure-shopware-6/01.en.md]: :see_no_evil:

**Pull request merged**